packages/api-request-builder/src/delete.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+/**
+ * Set the `version` number to the internal state of the service instance
+ * in order to generate a delete uri
+ *
+ * @param  {number} version - The version of the item
+ * @throws if `version` is missing or not a number.
+ * @return {Object} The instance of the service, can be chained.
+ */
+
+export default function toDelete (version: number): Object {
+  if (!version || typeof version !== 'number')
+    throw new Error('Required version to delete missing or not valid')
```

Comment:
I wouldn't mention any "delete" thingy in this function. If we are considering `version` as just a query parameter, it can be used for anything, not just "delete".

So I would keep this generic. See also my other comment above.